tests/functional/elements.js

```diff
@@ -2,27 +2,234 @@
+ const generalElements = {     
     CONTROLLER_TITLE: "controllerTitle",
-    CITYLIST_VISIBLE: "cityListVisible",
-    VPN_MULTHOP_CHEVRON: 'vpnCollapsibleCardChevron',
-    VPN_COLLAPSIBLE_CARD: 'vpnCollapsibleCard',
+    CONTROLLER_SUBTITLE: 'controllerSubTitle',
+    CONTROLLER_TOGGLE: 'controllerToggle'
+}
+
+const elementState = {
+    COUNTRY_VIEW: "serverCountryView",
+    CITYLIST_VISIBLE: "cityListVisible",   
+}
+
+const navBar = {
+    NAVIGATION_BAR: 'navigationBar',
+    HOME: 'navigationLayout/navButton-home',
+    MESSAGES: 'navigationLayout/navButton-messages',
+    SETTINGS: 'navigationLayout/navButton-settings',
+}
+
+const telemetryView = {    
+    TELEMETRY_POLICY_BUTTON: 'telemetryPolicyButton',
+    DECLINE_TELEMETRY: 'declineTelemetryLink',
+    POST_AUTHENTICATION_BUTTON: 'postAuthenticationButton'    
+}
+
+/*
+    Settings screen and related screens
+*/
+const settingsView = {
+    SETTINGS: 'settingsView',    
+    SCREEN: 'settingsView-flickable', // or SETTINGS_VIEW: 'settingsView',
+    USER_PROFILE: 'settingsUserProfile-manageAccountButton',
+    TIPS_AND_TRICKS: 'settingsTipsAndTricks',
+    NETWORK_SETTINGS: 'settingsNetworking',
+    SYSTEM_PREFERENCE: 'settingsPreferences',
+    MY_DEVICE: 'settingsDeviceList',
+    GET_HELP: 'settingsGetHelp',
+    ABOUT_US: 'settingsAboutUs',
+    SIGN_OUT: 'settingsLogout',
+    BACK: 'settings-back',
+
+
+    // GET HELP VIEW
+    getHelpView: {
+        BACK: 'getHelpBack'
+    },
+
+    tipsAndTricksView: {
+        SCREEN: 'settingsTipsAndTricksPage',
+        BACK: 'tipsAndTricks-back'
+    },
+
+    networkSettingsView: {
+        SCREEN: 'settingsView-flickable',
+    },
+
+    systemPreferenceView: {
+        LANGUAGE: 'settingsLanguages',
+        NOTIFICATIONS: 'settingsNotifications',
+
+        languageSettingsView: {
+            systemLanguageToggle: 'settingsSystemLanguageToggle'
+        }
+    },
+
+    aboutUsView: {
+        SCREEN: 'viewAboutUs',
+        LIST: 'aboutUsList',
+        TOS: 'aboutUsList/aboutUsList-tos',
+        PRIVACY: 'aboutUsList/aboutUsList-privacy',
+        LICENSE: 'aboutUsList/aboutUsList-license'
+    },
+
+    myDevicesView: {
+        BACK: 'deviceList-back',
+        DEVICE_LIST: 'deviceListView',
+        DEVICE_LIMIT_HEADER: 'deviceLimitHeader',
+        CONFIRM_REMOVAL_BUTTON: 'confirmRemoveDeviceButton'
+    },
+
+    getHelpView: {
+        BACK: 'getHelpBack',
+        LINKS: 'getHelpLinks',
+        HELP_CENTER: 'helpCenter',
+        SUPPORT: 'inAppSupport',
+        LOGS: 'viewLogs',
+        FEEDBACK: 'settingsGiveFeedback',
+
+        giveFeedbackView: {
+            SCREEN: 'giveFeedbackView'
+        },
 
-    generateCountryId: (serverCode) => {
+        contactSupportView: {
+            USER_INFO: 'contactUs-userInfo',
+            UNAUTH_USER_INPUTS: 'contactUs-unauthedUserInputs'
+        }
+    }
+}
+
+/*
+    In-app messaging screen and related screens
+*/
+const inAppMessagingView = {
```

Comment:
in general, if it's in `src/ui/screens` it's a screen.

Reply:
that was actually the thought behind the "***View" naming pattern because every qml file in that dir is name "View****"